GOVERNANCE.md

```diff
@@ -0,0 +1,49 @@
+# Submariner Governance
+
+## Open Governance Checklist
+
+The opengovernance.dev project [provides the following
+checklist](https://github.com/opengovernance/opengovernance.dev/blob/master/README.md#checklist) for defining open governance.
+
+Submariner's answers are provided for each point.
+
+This is not meant to define Submariner's governance, but instead to document it clearly. Authoritative sources are cited inline.
+
+### Ownership
+
+* Who owns the copyright on contributed code?
+  * All Submariner code is linted to verify the copyright notices specify "Contributors to the Submariner project", per the
+    [CNCF recommendations](https://github.com/cncf/foundation/blob/master/copyright-notices.md#copyright-notices).
+    **TODO: [Update to this format](https://github.com/submariner-io/submariner/issues/1322)**
+* Who owns the domain(s) for the project?
+  * Submariner is a CNCF Sandbox project, so all Submariner domains are owned by CNCF.
+* Who owns the trademark for the project, is it neutrally owned and governed? Are there open trademark guidelines?
+  * Submariner is a CNCF Sandbox project, so all Submariner trademarks are owned by the CNCF.
+* How can users license the project’s branding? Are there open branding guidelines?
+  * Submariner is a CNCF Sandbox project, so please see the CNCF/artwork project's
+    [Trademark and Logo Policy](https://github.com/cncf/artwork#trademark-and-logo-policy).
+* If the project raises funds, who owns it?
+  * The Submariner Contributors. **TODO: Document this elsewhere.**
+
+### Process
+
+* Who makes decisions on how the project performs releases?
+  * Project Owners are responsible for defining milestones and releases, per the Submariner
+    [Contributor Roles](https://submariner.io/community/contributor-roles/).
+* How can the project contributors become committers?
+  * In short, by reviewing code and receiving committer/owner approval.
+    See the [Committer Requirements](https://submariner.io/community/contributor-roles/) for details.
+* How are project committers removed?
+  * By stepping down or majority vote of Project Owners. **TODO: Document this elsewhere.**
+* If the project raises funds, who decides how this money is spent?
+  * The Submariner Contributors. **TODO: Document this elsewhere.**
+* Who decides the project roadmap?
+  * The [roadmap planning process](https://submariner.io/community/roadmap/#planning-process) is public and open to everyone.
+* Who can participate in security disclosure issues?
+  * Project Owners or committers who are required for the process, and members of the submariner-security@googlegroups.com email list.
+    **TODO: Document this elsewhere.**
```

Comment:
Proposed this as authoritative docs: https://github.com/submariner-io/submariner-website/pull/518